src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -23,22 +24,22 @@
 public class SampleDataUtil {
     public static Person[] getSamplePersons() {
         return new Person[] {
-            new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
+            new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Pay("alexyeoh@example.com"),
```

Comment:
Not sure whether anything is affected by this, but maybe change to numbers for the Pay?

Reply:
Good catch! I'll fix that as it's used for the initial sample data.